<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>httpconnect</class>
 <widget class="QWidget" name="httpconnect">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>89</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="le_ipInput">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>151</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit{
	color: #00ADB5;
	background-color:#222831;
	border:1px solid #00ADB5;
	padding: 0px 0px 0px 10px;
}
</string>
   </property>
   <property name="text">
    <string>192.168.0.202</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="bn_connect">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
color: #222831;
background-color:#00ADB5;
border:transparent;
}
</string>
   </property>
   <property name="text">
    <string>connect</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: #222831;
background-color:#00ADB5;
border:transparent;
</string>
   </property>
   <property name="text">
    <string>WIFI SSID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="le_ssid">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>10</y>
     <width>161</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Bahnschrift Light</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">	color: #00ADB5;
	background-color:#222831;
border:1px solid #00ADB5;
	padding: 0px 0px 0px 0px;</string>
   </property>
   <property name="text">
    <string>CAM</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
